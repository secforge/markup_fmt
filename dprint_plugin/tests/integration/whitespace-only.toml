# Test whitespace-only content in custom blocks
[vue_custom_block]
default = "lang-attribute"
docs = "none"
metadata = "squash"
