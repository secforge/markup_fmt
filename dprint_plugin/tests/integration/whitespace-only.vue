<template>
  <div>Test whitespace-only content</div>
</template>

<!-- lang-attribute mode with whitespace-only (no lang, so preserved raw) -->
<i18n>


</i18n>

<!-- none mode with whitespace-only -->
<docs>


</docs>

<!-- squash mode with whitespace-only -->
<metadata>


</metadata>
