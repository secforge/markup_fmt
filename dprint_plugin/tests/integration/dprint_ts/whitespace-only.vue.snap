---
source: dprint_plugin/tests/integration.rs
assertion_line: 123
---
<template>
  <div>Test whitespace-only content</div>
</template>

<!-- lang-attribute mode with whitespace-only (no lang, so preserved raw) -->
<i18n>


</i18n>

<!-- none mode with whitespace-only -->
<docs>


</docs>

<!-- squash mode with whitespace-only -->
<metadata></metadata>
